### PR TITLE
Add 0.173 release notes

### PR DESCRIPTION
**Pull request by dain**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.173.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+=============
+Release 0.173
+=============
+
+General Changes
+---------------
+
+* Fix issue where selective aggregates for ``count`` with a constant argument were being ignored.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.173.rst]: The phrase "selective aggregates" isn't very understandable to the end user. Let's say
```
* Fix issue where ``FILTER`` was ignored for :func:`count` with a constant argument.

File: presto-docs/src/main/sphinx/release/release-0.173.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+* Support table comments for :doc:`create-table` and :doc:`create-table-as`.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.173.rst]: This is not the right syntax for links. See other release notes for example.

**Comment by dain** [comment]
@electrum updated

File: presto-docs/src/main/sphinx/release/release-0.173.rst (hunk 1 above)
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.173.rst]: Sure, but FWIW, that's the name of the feature (just like "common table expressions" vs WITH, or window functions vs OVER).

**Pull request closed without merging**